test/unit/gateways/payway_dot_com_test.rb

```diff
@@ -141,7 +141,7 @@ def test_successful_void_of_credit
   end
 
   def test_invalid_login
-    @gateway2 = PaywayDotComGateway.new(login: '', password: '', company_id: '')
+    @gateway2 = PaywayDotComGateway.new(login: '', password: '', company_id: '', source_id: '')
```

Comment:
@dsmcclain out of interest I looked at what you get if you completed remove `source_id` and it looks like it's a different result from this one - `ArgumentError: Missing required parameter: source_id`

Can you create a test for that specifically and resubmit? Thanks!